package.json

```diff
@@ -31,7 +31,7 @@
     "@asyncapi/openapi-schema-parser": "^2.0.1",
     "@asyncapi/parser": "^1.15.1",
     "@asyncapi/react-component": "^1.0.0-next.38",
-    "@asyncapi/specs": "^3.0.0",
+    "@asyncapi/specs": "^2.14.0",
```

Comment:
I don't think I understand why you're downgrading to a previous version of schemas. Can you elaborate on this?

Reply:
Doesn't anyone read the PR description? 😄 

> switch to old json schemas due to problem that monaco cannot fetch schemas from https://asyncapi.com/definitions...

However I try to make a workaround to support latest specs.

Reply:
I did read the PR description but I still don't understand the reason.

Reply:
Look on latest changes.